index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Pumpkin Dash</title>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+</head>
+<body>
+<main>
+    <section id="titleSection" class="topSection"><h2>Pumpkin Dash</h2></section>
+    <section id="gameBox" class="middleSection">
+        <div id="startButton" class="button"> Start Game  </div>
+
+        <div id="playScreen" class="gameScreen">
+            <!--This div is left for celebrations etc-->
+            <div class="height30Percent"></div>
+            <!--This div is space for a possible cpu-->
+            <div class="height30Percent"></div>
+            <div class="height30Percent">
+                <div id="rightProfile" class="rightLeg character"></div>
+                <div id="leftProfile" class="leftLeg character"></div>
+                <div class="pumpkin"></div>
+                <div id="pumpkinTextBox" class="pumpkinFinishText"></div>
+            </div>
+        </div>
+    </section>
+    <div id="scoreBoard" class="scoreSection">
+        <div class="instructions">Instructions:
+             Race Pumpkin Reaper towards his next victim (giant pumpkin head).
+             Press keys 'a' and 's' alternately to move Pumpkin Reaper, as fast as you can.
+             Cross the finish line to win the game.</div>
+        <div id="timer" class="scoreBox">0:00</div>
```

Comment:
the timer is set to input the time so we can probably remove the string 0:00 as this will be populated on game start